Code/GraphMol/FilterCatalog/Filters.cpp

```diff
@@ -791,6 +791,906 @@ const FilterProperty_t ZINC_PROPS[] = {
 const unsigned int NUM_ZINC_PROPS =
     static_cast<unsigned int>(sizeof(ZINC_PROPS) / sizeof(FilterProperty_t));
 
+/////////////////////////////////////////////////////////////////////////////////////////
+// # Chembl 23 structural alert, rule set CHEMBL_Glaxo
+//
+const FilterData_t CHEMBL_Glaxo[] = {
```

Comment:
I think having these giant collections of SMARTS in this file makes it unmanageable.
I'd suggest going with:
`chembl_glaxo.in`, `chembl_dundee.in`, etc.
(yeah, I know we have other definitions in here too, but we could think about moving them into their own `.in` files in a separate PR

Reply:
I also noted that the Filters.cpp.in was unused.  I can remove it in this PR or we can merge this and make a separate one for splitting out the catalogs and removing it.

Reply:
> I also noted that the Filters.cpp.in was unused.  I can remove it in this PR or we can merge this and make a separate one for splitting out the catalogs and removing it.

I think it's worth removing it here and then doing a separate PR to pull the filters out

Reply:
@greglandrum it was easy enough to split up.